index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <meta name="description" content="Weather Report">
+        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"> 
+        <title>Weather</title>
+        <link rel="stylesheet" type="text/css" href="styles/index.css">
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.3/css/all.min.css">
+    </head>
+    <body>
+        <h1 class="title">Weather Report</h1>
+        <p class=cityNameDisplay>For the lovely city of San Francisco`</p>
```

Comment:
missing quotes around the class name 
```suggestion
        <p class="cityNameDisplay">For the lovely city of San Francisco`</p>
```